lib/HTML/HTMLFormat.php

```diff
@@ -44,6 +47,28 @@ public function getDirectives(): array
         ];
     }
 
+    /** @return TextRole[] */
+    public function getTextRoles(): array
+    {
+        return [
+            new WrapperTextRole('aspect', '<em class="aspect">%s</em>'),
+            new WrapperTextRole('command', '<strong class="command">%s</strong>'),
+            new WrapperTextRole('dfn', '<em class="dfn">%s</em>'),
+            new WrapperTextRole('file', '<span class="pre file">%s</span>'),
+            new WrapperTextRole('guilabel', '<span class="guilabel">%s</span>'),
+            new WrapperTextRole('kbd', '<kbd class="kbd docutils literal notranslate">%s</kbd>'),
+            new WrapperTextRole('mailheader', '<em class="mailheader">%s</em>'),
+            new WrapperTextRole('math', '<math>%s</math>'),
+            new WrapperTextRole('emphasis', '<em>%s</em>'),
+            new WrapperTextRole('literal', '<span class="pre">%s</span>'),
+            new WrapperTextRole('strong', '<em class="strong">%s</em>'),
+            new WrapperTextRole('subscript', '<sub>%s</sub>', ['sub']),
+            new WrapperTextRole('superscript', '<sup>%s</sup>', ['sup']),
+            new WrapperTextRole('title-reference', '<cite>%s</cite>', ['t', 'title']),
+            new DefinitionTextRole('abbreviation', '<abbr title="%2$s">%1$s</abbr>', ['abbr']),
```

Comment:
Where does the HTML representation of the roles come from? Did I miss something in the links?

Reply:
RST does not specify any HTML representations as RST can be transformed into many different Formats. So I took those we are doing in TYPO3 or you have a better idea?

Reply:
Most of the tags make sense as they are the equivalent of the role. I'm confused about `<em class="strong">%s</em>` for strong. I'd expect the `<strong>` tag.

Reply:
thats true, changed it to stron